components/chef-ui-library/src/molecules/chef-phat-radio/chef-phat-radio.tsx

```diff
@@ -22,6 +22,9 @@ export class ChefPhatRadio {
    * The value of the currently toggled option.
    */
   @Prop({ mutable: true }) value = '';
+  // canDeselect is an optional property that when true allows users
+  // to deselect all options by clicking on the currently selected option.
+  @Prop() canDeselect: boolean;
```

Comment:
While I'm rambling, may as well nitpick on some more wording. I'd replace `clicking` with a more generic verb since radio options are keyboard selectable as well.

```
An optional property that when set allows users to deselect
an option by [selecting,toggling,etc] a currently selected option.
```

Reply:
+1 to not using the word _clicking_ we removed that from our public-facing docs as well

Reply:
I'm on board with deselectable.  Your reasoning makes good sense and i've no strong opinions either way. 👍

Reply:
Thanks for the tip on the auto generating docs too.  Those are great!